src/test/java/org/fogbowcloud/manager/core/OrderControllerTest.java

```diff
@@ -179,17 +188,22 @@ public void testGetResourceInstance() throws Exception {
         orderInstance.setState(InstanceState.READY);
         order.setInstanceId(instanceId);
 
-        doReturn(orderInstance).when(localCloudConnector).getInstance(Mockito.any(Order.class));
+        Mockito.doReturn(orderInstance).when(localCloudConnector).getInstance(Mockito.any(Order.class));
 
         PowerMockito.mockStatic(CloudConnectorFactory.class);
-        given(CloudConnectorFactory.getInstance()).willReturn(cloudConnectorFactory);
+        BDDMockito.given(CloudConnectorFactory.getInstance()).willReturn(cloudConnectorFactory);
 
+        //exercise
         Instance instance = this.ordersController.getResourceInstance(order);
+
+        // verify
```

Comment:
expected and actual seems swapped. please, @llvieira check the whole PR to find other errors like that